traffic_ops/testing/api/v4/deliveryservices_test.go

```diff
@@ -1252,3 +1254,42 @@ func GetTestDeliveryServicesURLSigKeys(t *testing.T) {
 		t.Error("failed to get url sig keys: " + err.Error())
 	}
 }
+
+func CreateTestDeliveryServicesURLSigKeys(t *testing.T) {
+	if len(testData.DeliveryServices) == 0 {
+		t.Fatal("couldn't get the xml ID of test DS")
+	}
+	firstDS := testData.DeliveryServices[0]
+	if firstDS.XMLID == nil {
+		t.Fatal("couldn't get the xml ID of test DS")
+	}
+
+	_, _, err := TOSession.CreateDeliveryServiceURLSigKeys(*firstDS.XMLID, nil)
+	if err != nil {
+		t.Error("failed to create url sig keys: " + err.Error())
+	}
+
+	newKeys, _, err := TOSession.GetDeliveryServiceURLSigKeys(*firstDS.XMLID, nil)
+	if err != nil {
+		t.Error("failed to get url sig keys: " + err.Error())
+	}
+	if len(newKeys) == 0 {
+		t.Errorf("failed to create url sig keys")
+	}
```

Comment:
For this test we should call `Create...` again and `Get...` again and make sure the two generated keys are different. This will also ensure that `Create...` is idempotent.